File: Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if not nums:
            return 0
        length = 1
        for i in range(len(nums)-1):
            if nums[i+1] != nums[i]:
                nums[length] = nums[i+1]
                length += 1
        return length

File: Top Interview Questions/Easy Collection/C/Array/Remove Duplicates from Sorted Array.c
int removeDuplicates(int* nums, int numsSize){
    if(numsSize == 0)
        return 0;
        
    int cur = nums[0];
    int next_idx = 1;
    for(int i=1; i<numsSize; i++){
        if(nums[i] != cur){
            nums[next_idx] = nums[i];
            cur = nums[i];
            next_idx++;
        }
    }
    return next_idx;
}